games-action/openclonk-7.0

```diff
@@ -0,0 +1,118 @@
+SUMMARY="A free multiplayer action game where you control clonks"
+DESCRIPTION="OpenClonk is a free multiplayer action game in which you\
+control clonks, small, but witty and nimble humanoid beings. The game\
+is mainly about mining, settling and fast-paced melees."
+HOMEPAGE="http://www.openclonk.org/"
+COPYRIGHT="2009-2016 The OpenClonk Team and contributors"
+LICENSE="ISC"
+SOURCE_URI="http://www.openclonk.org/builds/release/$portVersion/openclonk-$portVersion-src.tar.bz2"
+CHECKSUM_SHA256="bc1a231d72774a7aa8819e54e1f79be27a21b579fb057609398f2aa5700b0732"
+SOURCE_DIR="openclonk-$portVersion-src"
+
+ARCHITECTURES="x86_gcc2 ?x86 ?x86_64"
+
+PROVIDES="
+	openclonk$secondaryArchSuffix = $portVersion
+	app:OpenClonk$secondaryArchSuffix = $portVersion
+	application$secondaryArchSuffix = $portVersion
+	bundled$secondaryArchSuffix = $portVersion
+	mimehandler$secondaryArchSuffix = $portVersion
+	openclonk(x86-64)$secondaryArchSuffix = $portVersion
+        "
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libpng$secondaryArchSuffix
+	lib:libjpeg_turbo$secondaryArchSuffix
+	lib:libfreetype$secondaryArchSuffix
+	lib:libogg$secondaryArchSuffix
+	lib:libvorbis$secondaryArchSuffix
+	lib:libxrandr$secondaryArchSuffix
+	lib:libGL$secondaryArchSuffix
+	lib:libGLEW$secondaryArchSuffix
+	lib:libGLU$secondaryArchSuffix
+	lib:libault$secondaryArchSuffix
+	lib:libat-1.0k$secondaryArchSuffix
+	lib:libc$secondaryArchSuffix
+	lib:libcairo-gobject$secondaryArchSuffix
+	lib:libcairo$secondaryArchSuffix
+	lib:libgcc_s$secondaryArchSuffix
+	lib:libgdk_pixbuf-2.0$secondaryArchSuffix
+	lib:libgio-2.0$secondaryArchSuffix
+	lib:libglib-2.0$secondaryArchSuffix
+	lib:libgobject-2.0$secondaryArchSuffix
+	lib:libgthread-2.0$secondaryArchSuffix
+	lib:libgtk-3$secondaryArchSuffix
+	lib:libixml$secondaryArchSuffix
+	lib:libm$secondaryArchSuffix
+	lib:libopenal$secondaryArchSuffix
+	lib:libpango-1.0$secondaryArchSuffix
+	lib:libpangocairo-1.0$secondaryArchSuffix
+	lib:libpng16$secondaryArchSuffix
+	lib:libpthread$secondaryArchSuffix
+	lib:librt$secondaryArchSuffix
+	lib:libstdc++$secondaryArchSuffix
+	lib:libthreadutil$secondaryArchSuffix
+	lib:libtinyxml$secondaryArchSuffix
+	lib:libupnp$secondaryArchSuffix
+	lib:libvorbisfile$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	lib:zlib$secondaryArchSuffix
```

Comment:
It's lib:libz not lib:zlib
https://github.com/haikuports/haikuports/blob/99b3c76c3e8178c62044b9f0e0594db0b1ff518c/sys-libs/zlib/zlib-1.2.11.recipe#L26